common/app/model/Asset.scala

```diff
@@ -42,7 +42,7 @@ object ImageAsset {
 }
 
 case class ImageAsset(
-  index: Int,
+  index: Int = 0,
```

Comment:
Don't forget to remove `= 0` from here too https://github.com/guardian/frontend/blob/master/common/app/model/ImageOverride.scala#L24